main.py

```diff
@@ -1,9 +1,185 @@
-print("Wyświetlenie listy zrobionych zadań: 1")
-print("Wyświetlenie listy zadań do wykonania: 2")
-print("Dodaj zadanie: 3")
-while True:
-    odpowiedz = input('Wybierz menu ')
-    if odpowiedz == "3":
-        nowe_zadanie = input("Napisz jakie zadanie chcesz dodać: ")
-        print(nowe_zadanie)
-        break
\ No newline at end of file
+import sqlite3
+from sqlite3 import Error
```

Comment:
Jeśli importujesz już całego liba (linia 1) to nie musisz potem pojedynczych elementów z tego liba wciągać. W miejscu wywołania Error, musisz przed dodać sqlite3. czyli `Error` zamieniasz na `sqlite3.Error`

Reply:
OK rozumiem, poprawiono